cloudnative-pg.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: cloudnative-pg
+  version: 1.23.2
+  epoch: 0
+  description: CloudNativePG is a comprehensive platform designed to seamlessly manage PostgreSQL databases
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cloudnative-pg/cloudnative-pg
+      tag: v${{package.version}}
+      expected-commit: 4bef841286ef84c4b435a6204a7ee632e1d4fb57
+
+  - uses: go/build
+    with:
+      modroot: ./cmd/manager
+      output: manager
+      packages: .
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-plugins
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/kubectl-cnpg
+          output: plugins
+          subpackage: true
+
+  - name: ${{package.name}}-compat
```

Comment:
Hmm... Is the `/manager` path actually needed? Is setting the entrypoint to `/usr/bin/manager` sufficient? We should only create compat packages when we're sure we'll need them :)

Reply:
Sure. I'll just remove it for now.